gcloud/test_connection.py

```diff
@@ -374,7 +395,11 @@ def __init__(self, authorized=None):
 
     def authorize(self, http):
         self._called_with = http
-        return self._authorized
+
+        if callable(self._authorized):
```

Comment:
Would that matter though, considering we overwrite `_authorized` in the constructor? Maybe I'm just not thinking 100% on Monday morning. :|